splunk_topology/conf.yaml.example

```diff
@@ -35,9 +35,37 @@ init_config:
 # It is possible to specify multiple saved_searches on a single instance.
 instances:
   - url: "https://localhost:8089"
+
+    # username: "admin" ## deprecated; use basic_auth.username under authentication section
+    # password: "admin" ## deprecated; use basic_auth.password under authentication section
+
     # verify_ssl_certificate: false
-    username: "admin"
-    password: "admin"
+
+    ## Integration supports either basic authentication or token based authentication.
+    ## Token based authentication is preferred before basic authentication.
+    authentication:
+      basic_auth:
+        username: "admin"
+        password: "admin"
+      # token_auth:
+        ## Token for the user who will be using it
+        # name: "api-user"
+
+        ## it's first initial valid token which will be exchanged with new generated token as soon as the check starts
```

Comment:
"it's first initial valid token.." -> "The initial valid token.."?

Reply:
done.